connectors/es/index.py

```diff
@@ -127,3 +127,14 @@ async def get_all_docs(self, query=None, sort=None, page_size=DEFAULT_PAGE_SIZE)
             if count >= total:
                 break
             offset += len(hits)
+
+    async def meta(self):
+        response = await self.client.indices.get_mapping(index=self.index_name)
+        # Assume index name is the same pattern and the latest index is always the one with the largest suffix
+        latest_index = max(response.keys())
+        return response[latest_index].get("mappings", {}).get("_meta")
```

Comment:
I vaguely remember, that calls to `get_mappings` are very expensive on loaded clusters.

How often do we intend to do the calls?

Reply:
About every one hour (by the default interval)



> I vaguely remember, that calls to `get_mappings` are very expensive on loaded clusters.

This is new to me, 🤔  `get_mappings` will only try to get the mappings of the index, not the docs.